src/main/java/seedu/address/model/appointment/exceptions/AppointmentNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package seedu.address.model.appointment.exceptions;
+
+/**
+ * Signals that the operation is unable to find the specified person.
+ */
+public class AppointmentNotFoundException extends RuntimeException {
```

Comment:
I like how you added custom exception classes throughout the program! I noticed this across the other classes too. Good job Archit.